user/id.go

```diff
@@ -23,6 +23,14 @@ type ID struct {
 	w []byte
 }
 
+// WithScriptHash is a constructor for ID, using [util.Uint160].
+func WithScriptHash(scriptHash util.Uint160) ID {
```

Comment:
```suggestion
func NewFromScriptHash(scriptHash util.Uint160) ID {
```
and mark this refs #483, and mark `SetScriptHash` as `Deprecated`